util/reddit_scraper.py

```diff
@@ -0,0 +1,51 @@
+import random
+import config
+import asyncpraw
+
+# Gets a random post from POST_LIMIT posts on hot
+POST_LIMIT = 15
+USER_AGENT = 'Mozilla/5.0 (X11; CrOS x86_64 8172.45.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.64 Safari/537.36'
+
+async def get_aita():
+    r = asyncpraw.Reddit(user_agent=USER_AGENT, 
+                    client_id=config.REDDIT_ID, 
+                    client_secret=config.REDDIT_SECRET)
+    aita = await r.subreddit('AmItheAsshole')
+    posts = [post async for post in aita.hot(limit=POST_LIMIT)]
+    post = posts[random.randint(0, POST_LIMIT)]
+    while (post.stickied): post = posts[random.randint(0, POST_LIMIT)]
```

Comment:
Potential infinite loop?

Reply:
You can only have 2 pinned threads at once so no.